Filesystem/Filesystem.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Filesystem;
+
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem as BaseFilesystem;
+
+class Filesystem extends BaseFilesystem
+{
+    public function getContents($filepath)
```

Comment:
Once you stop mocking the finder (which should not be done if you want the test to actually test that things are working), you won't need to mock getting the file content either